sbt-bridge/src/xsbt/ConsoleInterface.scala

```diff
@@ -51,6 +33,12 @@ class ConsoleInterface {
       } ++
       Array("-classpath", classpathString)
 
-    new ReplDriver(completeArgs, classLoader = Some(loader)).runUntilQuit()
+    val driver = new ReplDriver(completeArgs, classLoader = Some(loader))
+
+    val s0 = driver.bindValues((bindNames, bindValues).zipped)
+    val s1 = driver.run(initialCommands)(s0)
+    // TODO handle failure during initialisation
```

Comment:
We will need to open issues for this and value bindings

Reply:
I have created #5069 to implement the bindings, but I don't understand how failure during initialization should be signaled. Should we mark it with some checked exception?

Reply:
Yes the current API that returns a state is not ideal. Let's not worry about it here, it can be fixed in a separate PR